layouts/index.html

```diff
@@ -111,54 +112,6 @@ <h2>Latest news</h2>
     </div>
 </div>
 
-<a name="download"></a>
```

Comment:
After removed the download table, the "Latest news" section looks a little bit odd.  I wonder if we could have the latest news (i.e. "GA Release 1.0.0 is available") and a link to "Posts Archive" there.  We may make this change later.
<img width="808" alt="image" src="https://user-images.githubusercontent.com/907380/106079405-1b67d680-6150-11eb-8202-c79041e4ab14.png">

Reply:
@szetszwo Sure, will do this later.